### PR TITLE
Enabling of all Alphafold versions to run with GPU for pulsar-cz

### PR DESCRIPTION
**Pull request by martindemko**

### CODE REVIEW

**Comment by kysrpex** [comment]
I need help. Why was the previous situation preventing some Alphafold versions from running with GPU?

**Comment by kysrpex** [comment]
Can you wait for #908 to be merged? It makes sense to rebase this one on top of it.

File: files/galaxy/tpv/destinations.yml.j2 [hunk 1]
```diff
@@ -286,9 +286,7 @@ destinations:
       - if: tool.id.startswith("toolshed.g2.bx.psu.edu/repos/galaxy-australia/alphafold2/alphafold") and helpers.tool_version_gte(tool, "2.1.2+galaxy0") and helpers.tool_version_lt(tool, "2.3")
         env:
           ALPHAFOLD_DB: "/storage/brno11-elixir/projects/alphafold/alphafold.db-2.2"
-        params:
-          singularity_volumes: "$job_directory:ro,$tool_directory:ro,$job_directory/outputs:rw,$working_directory:rw,/cvmfs/data.galaxyproject.org:ro,$SCRATCHDIR,/storage/praha5-elixir/home/galaxyeu:/home/galaxyeu,$ALPHAFOLD_DB:/data/db/databases/alphafold_databases/2.2:ro"
-      - if: tool.id.startswith("toolshed.g2.bx.psu.edu/repos/galaxy-australia/alphafold2/alphafold") and helpers.tool_version_gte(tool, "2.3")
+      - if: tool.id.startswith("toolshed.g2.bx.psu.edu/repos/galaxy-australia/alphafold2/alphafold") and helpers.tool_version_gte(tool, "2.3.1+galaxy2")
```
Comment (kysrpex) [inline files/galaxy/tpv/destinations.yml.j2:289]: Any reason not to stick with `helpers.tool_version_gte(tool, "2.3")`? If you do not, you are excluding versions `2.3.1+galaxy0` and `2.3.1+galaxy1`.

File: files/galaxy/tpv/destinations.yml.j2 [hunk 2]
```diff
@@ -286,9 +286,7 @@ destinations:
       - if: tool.id.startswith("toolshed.g2.bx.psu.edu/repos/galaxy-australia/alphafold2/alphafold") and helpers.tool_version_gte(tool, "2.1.2+galaxy0") and helpers.tool_version_lt(tool, "2.3")
         env:
           ALPHAFOLD_DB: "/storage/brno11-elixir/projects/alphafold/alphafold.db-2.2"
-        params:
-          singularity_volumes: "$job_directory:ro,$tool_directory:ro,$job_directory/outputs:rw,$working_directory:rw,/cvmfs/data.galaxyproject.org:ro,$SCRATCHDIR,/storage/praha5-elixir/home/galaxyeu:/home/galaxyeu,$ALPHAFOLD_DB:/data/db/databases/alphafold_databases/2.2:ro"
```
Comment (kysrpex) [inline files/galaxy/tpv/destinations.yml.j2:290]: I assume you have things set up in a way that the params from the destination itself will work.

File: files/galaxy/tpv/destinations.yml.j2 (hunk 1 above)
Comment (martindemko) [reply files/galaxy/tpv/destinations.yml.j2:289]: That's the purpose cause, based on the tool version, there is a difference in command; the two mentioned versions get DB data from /data path, and the higher versions get the data from /data/2.3 path. That's why I need to bind the database differently.

File: files/galaxy/tpv/destinations.yml.j2 (hunk 2 above)
Comment (martindemko) [reply files/galaxy/tpv/destinations.yml.j2:290]: Not exactly sure what you are trying to point out here

**Comment by martindemko** [comment]
Can you estimate how long it will take to merge [#919](https://github.com/usegalaxy-eu/infrastructure-playbook/pull/919)?

File: files/galaxy/tpv/destinations.yml.j2 (hunk 1 above)
Comment (kysrpex) [reply files/galaxy/tpv/destinations.yml.j2:289]: Oh yeah that is true, thanks for clarifying it. Sorry, I should have remembered than even I had to it like this [in the past](https://github.com/kysrpex/usegalaxy-eu-ansible/blob/41161cd7a001fdb94406473bc52253d11fed290a/files/galaxy/tpv/tools.yml#L143-L148).

File: files/galaxy/tpv/destinations.yml.j2 (hunk 2 above)
Comment (kysrpex) [reply files/galaxy/tpv/destinations.yml.j2:290]: Nothing really, I believe this comment was not needed.

**Comment by kysrpex** [comment]
> Can you estimate how long it will take to merge [this](https://github.com/usegalaxy-eu/infrastructure-playbook/pull/919)?

It was merged already. I also changed what had to be changed in your PR :)

**Review by kysrpex — APPROVED** [review]

**Comment by martindemko** [comment]
Great! Thanks. Although I'm afraid it was not the last time I had to bother you guys as to set Alphafold properly is really pain in my ... :)

**Comment by kysrpex** [comment]
> Great! Thanks. Although I'm afraid it was not the last time I had to bother you guys as to set Alphafold properly is really pain in my ... :)

It is not your fault. There should be a standard way to let the administrators of Pulsar clusters do this on their own, but it not the case.

**Pull request merged**